cime_config/testdefs/testmods_dirs/clm/rm_individual_lunits/user_nl_clm

```diff
@@ -0,0 +1,6 @@
+toosmall_soil = 5.0
```

Comment:
I think this same test should also set n_dom_landunit and n_dom_pft so we have a system test that uses all of them together. And you might as well also test collapse_urban in this system test at the same time. This might mean the name of the test directory should be expanded.

Reply:
Ok, combining all these into one test.

Reply:
Good call! Triggers an error now. Looking into it.

Reply:
Corrected. I will commit and push work this far and add the new unit test after.